### PR TITLE
[LTD-2241] Line numbers

### PR DESCRIPTION
**Pull request by alixedi**

### CODE REVIEW

File: caseworker/tau/views.py [hunk 1]
```diff
@@ -26,7 +26,10 @@ def queue_id(self):
 
     @cached_property
     def case(self):
-        return get_case(self.request, self.case_id)
+        case = get_case(self.request, self.case_id)
+        for (i, good) in enumerate(case.goods):
```
Comment (saruniitr) [inline caseworker/tau/views.py:30]: may be `start=1`?

File: caseworker/tau/templates/tau/good_detail.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% load advice_tags tau_tags %}
 <details class="govuk-details govuk-!-margin-bottom-0" data-module="govuk-details" style="text-align: left;">
     <summary class="govuk-details__summary">
-        <span class="govuk-details__summary-text">{{ good.good.name }}</span>
+        <span class="govuk-details__summary-text">{{ good.line_number }}.  {{ good.good.name }}</span>
```
Comment (saruniitr) [inline caseworker/tau/templates/tau/good_detail.html:4]: looks like there is more than one space after the dot unless that is the requirement

**Review by saruniitr — APPROVED** [review]
one small comment otherwise looks good to me

File: caseworker/tau/templates/tau/good_detail.html (hunk 2 above)
Comment (alixedi) [reply caseworker/tau/templates/tau/good_detail.html:4]: It looked better with 2 spaces lol

**Pull request merged**